lib/routes/index.ts

```diff
@@ -16,6 +16,6 @@
 
 import wifi from './wifi';
 
-export default {
+export {
```

Comment:
Why have an extra file for this export, and not just require `routes/wifi`?

Reply:
I made it so that we can extend this to ethernet networks (or any other type that can be managed through dbus NetworkManager), so that we could just include one module for both but yeah, ES6 to the rescue so that also can be done without a module including other modules.